src/domains/chart/components/chart-with-loader.tsx

```diff
@@ -47,31 +41,29 @@ export type Props = {
   attributes: Attributes
   chartUuid: string
   portalNode: HTMLElement
+  chartMetadata?: ChartMetadata
 }
 
 export const ChartWithLoader = ({
   attributes,
   chartUuid,
   portalNode,
+  chartMetadata,
 }: Props) => {
   /**
    * fetch chart details
    */
   const host = attributes.host || serverDefault
   const dispatch = useDispatch()
-  const selectChartMetadataRequest = useMemo(makeSelectChartMetadataRequest, [])
-  const { chartMetadata, isFetchingDetails } = useSelector((state: AppStateT) => (
-    selectChartMetadataRequest(state, { chartId: attributes.id, id: chartUuid })
-  ))
   useEffect(() => {
-    if (!chartMetadata && !isFetchingDetails) {
+    if (!chartMetadata) {
```

Comment:
i think we misunderstood - this component is used also by the Agent Dashboard, so we cannot make any breaking changes here, we just need to handle scenario when `chartMetadata` is passed as a prop. We need to handle `redux` still (for Dashboard), so we'll need to have those lines back (`isFetchingDetails`, `selectChartMetadataRequest`, etc.)